packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -191,26 +191,20 @@ MeetingInfoUtil.getDestinationType = async (from) => {
       return Promise.resolve(options);
     });
   } else if (hydraId.room) {
-    options.type = _CONVERSATION_URL_;
-    try {
-      await webex.internal.services.waitForCatalog('postauth');
-
-      const conversationUrl = webex.internal.conversation.getUrlFromClusterId({
-        cluster: hydraId.cluster,
-        id: hydraId.destination,
-      });
-
-      options.destination = conversationUrl;
-    } catch (e) {
-      LoggerProxy.logger.error(`Meeting-info:util#getDestinationType --> ${e}`);
-      throw e;
-    }
+    LoggerProxy.logger.error(
```

Comment:
Aren't we removing `hydraId.people` as well? I was under the impression that we were going to deprecate `hydraId` as a whole and not just `hydraId.people`

If not, I am curious to understand the reason behind the deprecation of Room ID.

CC - @arun3528

Reply:
@mkesavan13, I was also in doubt as to why only `hydraId.room` and not `hydraId.people`, Both types are different.
For Room ID one needs a license to start a meeting in space, that's why we want developers to create meetings before they come to the SDK,
Also, Person ID should be supported as it is, as we need to provide "PERSON_ID" type as part of the creation.

This is people - `"type": "direct"`
`Y2lzY29zcGFyazovL3VzL1BFT1BMRS81ZGJiMmRiNy1iMmMyLTQwMmItYTViOC03MWNkMzU3MDEwMjM`
If I decode this we can see 
`ciscospark://us/**PEOPLE**/5dbb2db7-b2c2-402b-a5b8-71cd35701023`


For RoomID  - `"type": "group",` `Y2lzY29zcGFyazovL3VzL1JPT00vNGFmOTRhYzAtNTg5Zi0xMWVlLTkzNjAtZmI3MGVlMWJkMzY2`
`ciscospark://us/ROOM/4af94ac0-589f-11ee-9360-fb70ee1bd366`

@arun3528, @rarajes2

Reply:
Ok. Make sense in that case.